wazimap_ng/profile/serializers/metrics_serializer.py

```diff
@@ -16,8 +16,11 @@ def absolute_value(profile_key_metric, geography):
     return MetricCalculator.absolute_value(data, profile_key_metric, geography)
 
 def subindicator(profile_key_metric, geography):
-    data = get_indicator_data(profile_key_metric, [geography]).first().data
-    return MetricCalculator.subindicator(data, profile_key_metric, geography)
+    indicator_data = IndicatorData.objects.filter(indicator__profilekeymetrics=profile_key_metric, geography=geography)
```

Comment:
`get_indicator_data` does exactly that.

```suggestion
    indicator_data = get_indicator_data(profile_key_metric, [geography])
```